docs/getting-started.md

```diff
@@ -10,7 +10,7 @@ If you prefer learning with an example, check out our [Hello World](./user-guide
 
 These are the platform requirements to use Authorino:
 
-- [**Kubernetes**](https://kubernetes.io) server (recommended v1.21 or later), with permission to create Kubernetes Custom Resource Definitions (CRDs) (for bootstrapping Authorino and Authorino Operator)
+- [**Kubernetes**](https://kubernetes.io) server (recommended v1.22 or later), with permission to create Kubernetes Custom Resource Definitions (CRDs) (for bootstrapping Authorino and Authorino Operator)
```

Comment:
Is this intentional or just from a replace all? 🤔

Reply:
Oops. Nice catch! My bad. Fixing it now.

Reply:
Done.